custom/shared/src/test/java/com/solarwinds/opentelemetry/extensions/SpanStacktraceFilterTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * © SolarWinds Worldwide, LLC. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.solarwinds.opentelemetry.extensions;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import io.opentelemetry.sdk.trace.ReadableSpan;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class SpanStacktraceFilterTest {
+
+  @InjectMocks private SpanStacktraceFilter tested;
+
+  @Mock private ReadableSpan readableSpanMock;
+
+  @Test
+  void returnTrueWhenAttributeHasValue() {
+    when(readableSpanMock.getAttribute(any())).thenReturn("test-value");
```

Comment:
Appreciate the test! And empty string value would also return true (which would make sense since that's allowed by OTel), correct?

Reply:
yes